lib/routes/api/admin/index.js

```diff
@@ -10,8 +10,16 @@ var router = express.Router();
 
 router.use(auth.requireAdminOr401);
 
-/*
- * GET /admin/users
+/**
+ * @api {get} /admin/users Get All Users
+ * @apiPermission SiteAdmin
+ * @apiDescription Retrieves a list of all Strider users.
+ * @apiName AdminGetUsers
```

Comment:
Sure, I only did that assuming that a non-admin variation of `GetUsers` might exist someday, and just wanted to leave room for it.  I have no opinion beyond that and can remove that if you'd like.